src/routes/Home/index.tsx

```diff
@@ -6,13 +6,19 @@ import Page from 'src/components/layout/Page'
 import Row from 'src/components/layout/Row'
 import Col from 'src/components/layout/Col'
 
+import AddSafeWidget from 'src/components/Dashboard/AddSafe'
+import CreateSafeWidget from 'src/components/Dashboard/CreateSafe'
```

Comment:
These two share 90% of the code but I didn't want to refactor them at this time.